components/html-block/README.md

```diff
@@ -0,0 +1,21 @@
+# HTML Block
+
+## d2l-html-block
+
+The `d2l-html-block` element is a web component for displaying user-authored HTML. It includes styles for headings, lists, anchors and other elements.  In addition, it provides integration with MathJax for rendering MathML.
+
+```html
+<script type="module">
+  import '@brightspace-ui/core/components/html-block/html-block.js';
+</script>
+
+<d2l-html-block>
```

Comment:
Not sure if it's worth providing a sentence about why the `<template>` is necessary and how things won't work without it? I could see people not really looking at this, going `<d2l-html-block>${myStuff}</d2l-html-block>` and being confused as to why it doesn't work.

Actually... what would happen there? Can we detect that and make sure it doesn't "accidentally" kind-of work?

Reply:
Yeah that's a good point.  I will update and do some tests around that scenario.